src/domain/signup/utils.ts

```diff
@@ -197,6 +219,33 @@ export const getUpdateSignupPayload = ({
   };
 };
 
+export const getCreateSignupsPayload = ({
+  formValues,
+  registration,
+  reservationCode,
+}: {
+  formValues: SignupGroupFormFields;
+  registration: Registration;
+  reservationCode: string;
+}): CreateSignupsMutationInput => {
+  const { contactPerson, signups: signupsValues } = formValues;
+
+  const signups: SignupInput[] = signupsValues.map((signupData, index) => ({
+    ...getSignupPayload({
+      formValues,
+      responsibleForGroup: index === 0,
```

Comment:
`responsible_for_group` is no longer used by the back-end / it doesn't exist there anymore

Reply:
Removed responsibleForGroup in contact person related PR but it still slipped here. Removed